pkg/resourcekeeper/gc.go

```diff
@@ -162,6 +172,13 @@ func (h *resourceKeeper) garbageCollect(ctx context.Context, cfg *gcConfig) (fin
 			return false, waiting, errors.Wrapf(err, "failed to garbage collect legacy resource trackers")
 		}
 	}
+
+	if !cfg.disableApplicationRevisionGC {
+		if err = gc.GarbageCollectApplicationRevisionResourceTrackers(ctx); err != nil {
+			return false, waiting, errors.Wrapf(err, "failed to garbage collect application revision resource trackers")
```

Comment:
failed to garbage collect application revision resource trackers -> failed to garbage collect application revision